src/plugins/vcNarrator.tsx

```diff
@@ -0,0 +1,334 @@
+                type: OptionType.SELECT,
+                description: "Narrator Voice",
+                options: getEnglishVoices().map(v => ({
+                    label: v.name,
+                    value: v.voiceURI,
+                    default: v.default
+                }))
+            },
+            volume: {
+                type: OptionType.SLIDER,
+                description: "Narrator Volume",
+                default: 1,
+                markers: [0, 0.25, 0.5, 0.75, 1],
+                stickToMarkers: false
+            },
+            rate: {
+                type: OptionType.SLIDER,
+                description: "Narrator Speed",
+                default: 1,
+                markers: [0.1, 0.5, 1, 2, 5, 10],
+                stickToMarkers: false
+            },
+            joinMessage: {
+                type: OptionType.STRING,
+                description: "Join Message",
+                default: "{{USER}} joined {{CHANNEL}}"
+            },
+            leaveMessage: {
+                type: OptionType.STRING,
+                description: "Leave Message",
+                default: "{{USER}} left {{CHANNEL}}"
+            },
+            moveMessage: {
+                type: OptionType.STRING,
+                description: "Move Message",
+                default: "{{USER}} moved to {{CHANNEL}}"
+            },
+            muteMessage: {
+                type: OptionType.STRING,
+                description: "Mute Message (only self for now)",
+                default: "{{USER}} Muted"
+            },
+            unmuteMessage: {
+                type: OptionType.STRING,
+                description: "Unmute Message (only self for now)",
+                default: "{{USER}} unmuted"
+            },
+            deafenMessage: {
+                type: OptionType.STRING,
+                description: "Deafen Message (only self for now)",
+                default: "{{USER}} deafened"
+            },
+            undeafenMessage: {
+                type: OptionType.STRING,
+                description: "Undeafen Message (only self for now)",
+                default: "{{USER}} undeafened"
+            }
+        };
+    },
+
+    settingsAboutComponent({ tempSettings: s }) {
+        const [hasVoices, hasEnglishVoices] = useMemo(() => {
+            const voices = speechSynthesis.getVoices();
+            return [voices.length !== 0, voices.some(v => v.lang.startsWith("en"))];
+        }, []);
+
+        const types = useMemo(
+            () => Object.keys(Vencord.Plugins.plugins.VcNarrator.options!).filter(k => k.endsWith("Message")).map(k => k.slice(0, -7)),
+            [],
+        );
+
+        let errorComponent: React.ReactElement | null = null;
+        if (!hasVoices) {
+            let error = "No narrator voices found. ";
+            error += navigator.platform?.toLowerCase().includes("linux")
+                ? "Install speech-dispatcher or espeak and run Discord with the --enable-speech-synthesis flag"
```

Comment:
The actual flag is `--enable-speech-dispatcher`.
```suggestion
                ? "Install speech-dispatcher or espeak and run Discord with the --enable-speech-dispatcher flag"
```